### DISCUSSION TITLE
java call libvips.so error

### DISCUSSION THREAD

**Discussion by zw20045500** [libvips/libvips#2392] [Q&A]
Hi,  I use java language by jni to call  libvips.so.     I want to call libvips.so to support webp and gif, according to guide to load dependency.  when i execute java program to deal webp or gif., it not works.   but png, jpg is ok   thank you
![图片](https://user-images.githubusercontent.com/30209916/129023315-428465c6-98ef-4ed6-8188-b6039f32b80e.png)

**Comment by jcupitt** [comment]
Hello @zw20045500,

It looks like your linker can't find the imagemagick library. You need to fix your install somehow.
